ecomstore/tests/backend/customers/test_create_customers_smoke.py

```diff
@@ -80,35 +83,94 @@ def test_create_customer_fail_for_existing_email():
     }
     rs_body = woo_helper.post("customers", params=payload, expected_status_code=400)
 
-    breakpoint()
+
     assert rs_body['code'] == 'registration-error-email-exists', f"Create customer with existing user response does not"\
                               f" have expected text. Expected:'registration-error-email-exists', Actual: {rs_body['code']}"
 
     assert rs_body['data']['status'] == 400, f"Unexpected status code in body of response"\
                                              f"Expected 400 actual: {rs_body['data']['status']}"
-
+@pytest.mark.customers
 @pytest.mark.tcid32
 @pytest.mark.qactcid6
 def test_create_customer_fail_when_no_password_provided():
     #get random email
     random_info = generate_random_email_and_password(email_prefix='bleutc')
-
+    email = random_info['email']
     # make customer api call without password in the payload
 
+    payload = {"email": email}
+
+    woo_api_utility = WooAPIUtility()
+
+    rs_body = woo_api_utility.post(wc_endpoint='customers', params=payload, expected_status_code=400)
+
+    expected_response = {'code': 'rest_missing_callback_param','data': {'params':['password'], 'status': 400},'message':'Missing parameter(s): password'}
+    assert rs_body == expected_response, f"Got unexpected error creating user without password." \
```

Comment:
I didnt do it this way. but sweet. I was comparing each field at a time. The way you did it is just fine too.